src/main/java/homeworks/Infrastructure/wdm/DefaultWebDriverManager.java

```diff
@@ -0,0 +1,14 @@
+package homeworks.Infrastructure.wdm;
+
+public class DefaultWebDriverManager implements WebDriverManager{
+
```

Comment:
in ConfigurationManager add new method getRunOn() wich will be working with envVar  and return (String) local/cloud/remote

inside getBrowser method call method getRunOn()  and then in switch create one of the factories depends on runOn var (LocalWebDriverFactory/RemoteWebDriverFactory/CloudWebDriverFactory). Then from factory call method .create() which will be create specific browser on specific env.